### PR TITLE
syncthing: Add runit for user

### PR DESCRIPTION
**Pull request by Bridouz**
- The user needs to edit `conf` and
add its username for syncthing to work

### CODE REVIEW

File: srcpkgs/syncthing/files/syncthing/run [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+[ -r conf && . ./conf
+
+export USERNAME
+export HOME
+
+exec 2>&1
+exec chpst -u "$USERNAME" syncthing 
+-logflags 0
```
Comment (Vaelatern) [inline srcpkgs/syncthing/files/syncthing/run]: And what is this line doing here?

**Comment by Vaelatern** [comment]
I'm opposed to this, and I use syncthing, with my own user-specific service.

**Comment by the-maldridge** [comment]
This is a high magic solution, and relies on a user doing the right thing on the first try.  This unfortunately has been shown to be an impossibility in the current universe in which Void is deployed.  As a result I'm closing this until we can get a universe with more reliable end users.

**Comment by maxice8** [comment]
I'm against, runit doesn't have a user service framework like systemd and setting one up is up to the user most of the time.

If you set like runit upstream recommends then the service will start even when the user isn't logged in.

**Comment by Bridouz** [comment]
Ok,

You can close #5430 then. I messed up with branches :)

**Pull request closed without merging**